tasks.py

```diff
@@ -136,7 +136,7 @@ def _detect_space(branch=None, yes=False):
 
 DEPLOY_RULES = (
     ('prod', _detect_prod),
-    ('stage', lambda _, branch: branch == 'master'),
+    ('stage', lambda _, branch: branch == 'release'),
```

Comment:
The release branches created by git-flow are prefixed with "release" but also include more text: e.g., "release/v0.1.1". This should probably be `lambda _, branch: branch.startswith('release')`.